Makefile

```diff
@@ -120,6 +122,16 @@ clean:
 	rm -rf $(DIST) pkg/pillar/Dockerfile pkg/qrexec-lib/Dockerfile pkg/qrexec-dom0/Dockerfile \
 	       images/installer.yml images/rootfs.yml
 
+yetus-image:
+	docker build -t $(YETUS_IMG) ./build-tools/src/yetus
+
+yetus: yetus-image
+	@echo Running yetus, results saved to $(YETUS_OUT)
+	@mkdir -p $(YETUS_OUT)
+	@echo "bootstrap" > $(YETUS_OUT)/bootstrap
+	docker run --rm -v $(PWD):/src -v $(YETUS_OUT):/tmp/yetus-out $(YETUS_IMG)
```

Comment:
After chatting with @aw-was-here yesterday I wanted to suggest that you may have better luck with CircleCI if you simply pass all of the ENV variables down to docker execution. E.g.:

docker run --rm -v $(PWD):/src -v $(YETUS_OUT):/tmp/yetus-out --env-file <(env) $(YETUS_IMG)

Or something along these lines @deitch

Reply:
We can do that, it feels "messy" though. Well, anything to get us started. Make it work, then make it better.

Reply:
Try this.